src/tribler-core/tribler_core/components/metadata_store/db/orm_bindings/tag.py

```diff
@@ -0,0 +1,15 @@
+from pony import orm
+from pony.orm import composite_key
+
+
+def define_binding(db):
+    class Tag(db.Entity):
+        id = orm.PrimaryKey(int, auto=True)
+        infohash = orm.Required(bytes)
+        tag = orm.Required(str)
+        creator_public_key = orm.Required(bytes)
+        creator_sign = orm.Required(bytes)
```

Comment:
**Nitpick**: it is better to name these `public_key` and `signature` for two reasons:
 * `sign` is a verb while `signature` is a noun
 * it is better to stick to the same naming scheme Channels use, for consistency

Reply:
Thank you.
Tables have been extracted from Channels.